src/planner/plan/plan_load.cpp

```diff
@@ -0,0 +1,19 @@
+#include "binder/bound_load_statement.h"
+#include "common/cast.h"
+#include "planner/operator/logical_load.h"
+#include "planner/planner.h"
+
+using namespace kuzu::binder;
+
+namespace kuzu {
+namespace planner {
+
+std::unique_ptr<LogicalPlan> Planner::planLoad(const BoundStatement& statement) {
```

Comment:
Can we rename this to `planLoadExtension`? Just to be more detailed, so it is clearly different from `loadFrom`.